src/sentry/static/sentry/app/components/actions/actionLink.tsx

```diff
@@ -1,79 +1,66 @@
 import React from 'react';
 import styled from '@emotion/styled';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 
-import Confirm from 'app/components/confirm';
+import ActionButton from './button';
+import ConfirmableAction from './confirmableAction';
 
-type DefaultProps = {
-  disabled: boolean;
-  shouldConfirm: boolean;
-};
+type ConfirmableActionProps = React.ComponentProps<typeof ConfirmableAction>;
 
-type ActionLinkProps = DefaultProps & {
+type Props = Omit<ConfirmableActionProps, 'onConfirm' | 'confirmText' | 'children'> & {
   title: string;
-  message?: React.ReactNode;
+  children?: React.ReactNode;
   onAction: () => void;
```

Comment:
```suggestion
  onAction: () => void;
  children?: React.ReactNode;
```

I'd really like to have a plugin that autofixes order automatically (optional to the bottom).

Reply:
me too

Reply:
me too :)